src/main/java/org/elasticsearch/rest/action/admin/indices/analyze/RestAnalyzeAction.java

```diff
@@ -73,9 +72,9 @@ public void handleRequest(final RestRequest request, final RestChannel channel,
         if (RestActions.hasBodyContent(request)) {
             XContentType type = RestActions.guessBodyContentType(request);
             if (type == null) {
-                if (text == null) {
-                    text = RestActions.getRestContent(request).toUtf8();
-                    analyzeRequest.text(text);
+                if (texts == null || texts.length == 0) {
+                    texts = new String[]{ RestActions.getRestContent(request).toUtf8() };
+                    analyzeRequest.text(texts);
```

Comment:
Good question.

Analyze API has three way to receive text.
1. request parameter
2. request body, raw text
3. request body, JSON parameter

This case is 2.
In this case, I think it is difficult which letter we assume a delimiter.
And we leave this case for backward compatibility.

Then, I think it's OK we don't support multi valued text for this case.